From 07359c5b7d7e530688ebea177e945b43538b4a3d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=81lvaro=20Fern=C3=A1ndez=20Rojas?= <noltari@gmail.com>
Date: Wed, 25 Mar 2020 20:58:35 +0100
Subject: [PATCH] rpi-eeprom-update: change default include path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
---
 rpi-eeprom-update | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/rpi-eeprom-update
+++ b/rpi-eeprom-update
@@ -6,8 +6,8 @@ set -e
 
 script_dir=$(cd "$(dirname "$0")" && pwd)
 
-if [ -f /etc/default/rpi-eeprom-update ]; then
-   . /etc/default/rpi-eeprom-update
+if [ -f /etc/default/rpi-eeprom-update-default ]; then
+   . /etc/rpi-eeprom-update-default
 fi
 
 LOCAL_MODE=0
